src/java.security.jgss/share/classes/sun/security/jgss/krb5/Krb5Context.java

```diff
@@ -706,6 +706,7 @@ public KerberosTicket run() throws Exception {
                                     tgt);
                         }
                         if (GSSUtil.useSubjectCredsOnly(caller)) {
+                            @SuppressWarnings("removal")
```

Comment:
It looks like you missed removing the @SuppressWarnings("removal") from the initSecContext method above.

Reply:
Oops. Fixed.